presto-docs/src/main/sphinx/connector/pinot.rst

```diff
@@ -32,6 +32,129 @@ with a different name (making sure it ends in ``.properties``). For
 example, if you name the property file ``sales.properties``, Presto
 will create a catalog named ``sales`` using the configured connector.
 
+Map Pinot Schema to Presto Schema
+---------------------------------
+
+In general Pinot schema to Presto schema mapping are pretty straight forward.
+By default, the data type mapping follows the table below.
+
+.. list-table::
+   :widths: 100 100
+   :header-rows: 1
+
+   * - Pinot Data Type
+     - Presto Data Type
+   * - INT
+     - INTEGER
+   * - LONG
+     - BIGINT
+   * - FLOAT
+     - DOUBLE
+   * - DOUBLE
+     - DOUBLE
+   * - BYTES
+     - VARBINARY
+   * - STRING
+     - VARCHAR
+
+Since Pinot defines each field as dimension, metric or time(date_time) field,
+it's possible to infer Presto data type ``DATE`` and ``TIMESTAMP``:
+
+- A Pinot ``TIME`` field with timeGranularity ``{ "TimeFormat":"EPOCH", "TimeUnit":"DAYS", "TimeUnitSize": 1 }`` could be map to a ``DATE`` type.
+- A Pinot ``TIME`` field with timeGranularity ``{ "TimeFormat":"EPOCH", "TimeUnit":"MILLISECONDS", "TimeUnitSize": 1 }`` could be map to a ``TIMESTAMP`` type.
+- A Pinot ``DATE_TIME`` field with format ``1:DAYS:EPOCH`` could be map to a ``DATE`` type.
+- A Pinot ``DATE_TIME`` field with format ``1:MILLISECONDS:EPOCH`` could be map to a ``TIMESTAMP`` type.
+
+There are a few configurations that control this behavior:
+
+* ``pinot.infer-date-type-in-schema``: This config is false by default.
+  Setting it to true will infer a Pinot ``TIME``/``DATE_TIME`` field to ``DATE`` in Presto if possible.
+
+* ``pinot.infer-timestamp-type-in-schema``: This config is false by default.
+  Setting it to true will infer a Pinot ``TIME``/``DATE_TIME`` field to ``TIMESTAMP`` in Presto if possible.
+
+Below is an example with config: ``pinot.infer-timestamp-type-in-schema=true``.
+
+Sample Pinot Schema:
+
+.. code-block:: JSON
+
+  {
+    "schemaName": "meetupRsvp",
+    "dimensionFieldSpecs": [
+      {
+        "name": "venue_name",
+        "dataType": "STRING"
+      },
+      {
+        "name": "event_name",
+        "dataType": "STRING"
+      },
+      {
+        "name": "event_id",
+        "dataType": "STRING"
+      },
+      {
+        "name": "event_time",
+        "dataType": "LONG"
+      },
+      {
+        "name": "group_city",
+        "dataType": "STRING"
+      },
+      {
+        "name": "group_country",
+        "dataType": "STRING"
+      },
+      {
+        "name": "group_id",
+        "dataType": "LONG"
+      },
+      {
+        "name": "group_name",
+        "dataType": "STRING"
+      }
+    ],
+    "metricFieldSpecs": [
+      {
+        "name": "rsvp_count",
+        "dataType": "INT"
+      }
+    ],
+    "timeFieldSpec": {
+      "incomingGranularitySpec": {
+        "name": "mtime",
+        "dataType": "LONG",
+        "timeType": "MILLISECONDS"
+      }
+    }
+  }
+
+Sample Presto Schema:
+
+.. code-block:: none
+
+  $ ./presto-cli --catalog pinot --schema default
```

Comment:
I'd prefer not putting commands in documentation. The example could just contains pinot schema and presto schema

Reply:
Sounds good, updated the doc.